cli/dev.js

```diff
@@ -126,6 +121,8 @@ function watchMould() {
                 return
             }
 
+            copyMould()
+
```

Comment:
symlink userland's mould dir -> editor

when importing react in resolvers.ts, will cause multi react instance issue